go/vt/schemamanager/tablet_executor.go

```diff
@@ -249,13 +249,9 @@ func (exec *TabletExecutor) executeSQL(ctx context.Context, sql string, execResu
 				return err
 			}
 			for _, onlineDDL := range onlineDDLs {
-				if exec.ddlStrategySetting.IsSkipTopo() {
-					exec.executeOnAllTablets(ctx, execResult, onlineDDL.SQL, true)
-					if len(execResult.SuccessShards) > 0 {
-						exec.wr.Logger().Printf("%s\n", onlineDDL.UUID)
-					}
-				} else {
-					exec.executeOnlineDDL(ctx, execResult, onlineDDL)
+				exec.executeOnAllTablets(ctx, execResult, onlineDDL.SQL, true)
+				if len(execResult.SuccessShards) > 0 {
+					exec.wr.Logger().Printf("%s\n", onlineDDL.UUID)
```

Comment:
The change here is to remove `if exec.ddlStrategySetting.IsSkipTopo() {` -- we've already hard coded that as `true` is previous releases.